### PR TITLE
Implement CVS/XLSX parse for enrollment report

### LINKED ISSUE #82: Implement ReportService.parse
Implement the method for parsing CSV or XSLX files into our data model

### PR DESCRIPTION
**Pull request by cailyncodes**
Should close #82 

Merge after #81

### CODE REVIEW

File: src/services/enrollment/EnrollmentService.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { FlattenedEnrollment, Enrollment } from '../../models';
```
Comment (cailyncodes) [inline src/services/enrollment/EnrollmentService.ts:1]: These methods are accidentally preemptive -- they'll probably be used after checking data and submitting to the backend to be persisted in the db

File: src/models/flattenedEnrollment.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+export interface FlattenedEnrollment {
```
Comment (cailyncodes) [inline src/models/flattenedEnrollment.ts:1]: We shouldn't have to save this to the db at any point -- it's just for the initial parse and convenience to the front end code with model generation

File: client/src/containers/Upload/Upload.tsx [hunk 3]
```diff
@@ -43,7 +43,7 @@ const Upload: React.FC = () => {
       })
       .then((value) => {
         setStatus({
-          filename: (value as any).filename,
+          reportId: value.id,
```
Comment (JordanGuinn) [inline client/src/containers/Upload/Upload.tsx:46]: Ahhhh, okay - this answers my question on the previous PR 💯👍🏽

File: src/services/report/ReportService.ts [hunk 4]
```diff
@@ -17,12 +17,26 @@ export class ReportService {
...
-      enrollments: [],
-    };
+  ): FlattenedEnrollment[] {
+    const fileData = readFile(enrollmentReportBodyParams.path);
+
+    const sheet = Object.values(fileData.Sheets)[0];
+
+    const startingRow = this.getStartingRow(sheet);
+
+    const parsedSheet = utils.sheet_to_json<FlattenedEnrollment>(sheet, {
```
Comment (JordanGuinn) [inline src/services/report/ReportService.ts:27]: And this just... works?  With both xlsx and CSV files?  🏆🏆 Jeez, this is the simplest thing ever

**Pull request merged**